### DISCUSSION TITLE
How to match train and test set atom order

### DISCUSSION THREAD

**Discussion by ankitmathanker** [deepmodeling/dpdata#806] [Q&A]
When trying to create a dpdata train and test set for a system with 4 atom type Pt, C, H, O, I see that atoms order in training set is PtCHO whereas the test set is of order OHCPt. Is there a way for me to specify the ordering so that both train and test set have same order of atoms.

**Comment by wanghan-iapcm** [comment]
It is not necessary to impose the same order in deepmd training and test data, if the type_maps for both datasets are set correctly.
